main.py

```diff
@@ -0,0 +1,92 @@
+from fastapi import FastAPI, Request
```

Comment:
I don't understand where the client is. The server code should be at [https://github.com/elad-rubinstein/SHEST](url), not this repo.